js/constants.js

```diff
@@ -2,9 +2,24 @@
 
 window.onerror = function(e, url, line){
```

Comment:
:+1: for Sentry!

I am not sure if it's necessary to add this. I believe Sentry adds this via `install()` on its own (not 100% sure, though).